build_image.sh

```diff
@@ -150,7 +150,15 @@ elif [ "$IMAGE_TYPE" = "aboot" ]; then
     if [ "$SONIC_ENABLE_IMAGE_SIGNATURE" = "y" ]; then
         TARGET_CA_CERT="$TARGET_PATH/ca.cert"
         rm -f "$TARGET_CA_CERT"
-        [ -f "$CA_CERT" ] && cp "$CA_CERT" "$TARGET_CA_CERT"
+
+        # If the ca certificate does not exist, the test certificate will be used to sign the image
+        if [ ! -f "$CA_CERT" ]; then
+            TEST_CERT_PATH=files/image_config/secureboot/test-certs
```

Comment:
add `local` ?

Reply:
Seems it cannot be changed to local, it only can be used in a function.